docs/web/supported-browsers-and-frameworks.mdx

```diff
@@ -21,6 +21,90 @@ However, there are some gotchas to be aware of as well as common errors that you
 your stack.  The issues vary depending on the tooling in your project.  This page explains some helpful workarounds
 with links to examples.
 
+## Fetch API `keepalive`
+
+Connect-Web does not provide any native support for the Fetch API `keepalive` optionat the moment, but since Connect-Web
+uses the Fetch API under the hood, there are a few options for managing it yourself:
+
+### Using an interceptor
+
+You can use an interceptor to set the value. This is useful when you want to selectively set the flag for certain
+conditions. Here is an example for setting it only for server-streaming requests:
+
+```ts
+const transport = createConnectTransport({
+  baseUrl: "https://demo.connectrpc.com",
+  interceptors: [
+    (next) => {
+      return async (request) => {
+        return await next({
+          ...request,
+          init: {
+            ...request.init,
+            keepalive: request.method.kind === MethodKind.ServerStreaming,
```

Comment:
Why do we enable `keepalive` for server streams?

Reply:
This was just a basic example pulled from a Slack answer. We can modify this to be whatever.

Reply:
The Slack answer was given out of context. The user was asking about support for keeping idle server-streams alive. 

The fetch option `keepalive` is completely unrelated to this problem, it serves a different purpose (making sure that telemetry calls still go through if the tab is closed). This example could easily mislead users into thinking it would help with streaming.